TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/NoteDetailFragment.kt

```diff
@@ -33,20 +35,22 @@ class NoteDetailFragment : Fragment() {
     enum class PaintColors { Red, Blue, Green, Yellow, Purple }
 
     private lateinit var drawView: PenDrawView
+    private var deleted = false
```

Comment:
is there a way this could be like a method parameter or something else? it probably doesn't matter but it seems like it's only used in the updateNoteContents method

or I guess, since it's trying to the get the INode/Note associated with something that has already been deleted (I think?), is the extra check necessary?

Reply:
on file deletion, the fragment changes which causes an onPause event that triggers an updateNoteContents call. the flag prevents updateNoteContents from being called on a deleted file (since it will cause the app to crash). I'm definitely willing to look into a better solution, this was the quickest fix I thought of

Reply:
no worries idk why i keep looking at the smallest details hahaha no need to change anything